### ISSUE TITLE
Could you tell me the number of your steps when training,and the loss of G,F,DY,DX at last?

### ISSUE THREAD

**Issue by VirgoWonsa** [vanhuyz/CycleGAN-TensorFlow#59]

**Comment by vanhuyz** [comment]
Depending on data, but I usually trained about 100k~200k steps.
I think loss values in GAN do not mean too much. 
In my cases, value of D is very close to 0, and value of G is close to 1 (seems discriminator is too smart and always win).
